### PR TITLE
add react-native-hapticlabs

### PR DESCRIPTION
**Pull request by robot-controller**
# 📝 Why & how

This PR adds a new library.

# ✅ Checklist

- [X ] Added library to **`react-native-libraries.json`**

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -12037,5 +12037,15 @@
...
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/HapticlabsIO/react-native-hapticlabs",
+    "npmPkg": "react-native-hapticlabs",
+    "examples": [
+      "https://github.com/HapticlabsIO/react-native-hapticlabs"
+    ],
+    "images": ["https://dxcgyjzgzwauhbaxbuqp.supabase.co/storage/v1/object/public/HL%20logo/HL%20icon.png"],
```
Comment (Simek) [inline react-native-libraries.json:12047]:
```suggestion
```

Images should show the lib functionality, and we do not accept branding materials:
* https://github.com/react-native-community/directory?tab=readme-ov-file#images

**Review by Simek — COMMENTED** [review]
Hello @robot-controller, thanks for adding new entry to the library! 👍 

Just got one suggestion according to the provided asset, otherwise LGTM!

File: react-native-libraries.json (hunk 1 above)
Comment (robot-controller) [reply react-native-libraries.json]: Thank you for pointing it out and my apologies for missing it in the guide.

Comment (Simek) [reply react-native-libraries.json]: No worries, thanks for the quick update! Can you also rebase your PR?

Comment (robot-controller) [reply react-native-libraries.json]: Done; sorry that I accidentally broke my branch, but it should be all ready now.

**Review by Simek — APPROVED** [review]
Thanks! 👍

**Pull request merged**